deploy/kubernetes/examples/demo-pvc-clone-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demo-pvc-cloned-pvc
+spec:
+  volumeMode: Filesystem
+  accessModes:
+  - ReadWriteOnce
```

Comment:
Same access modes that supported on regular pvc?

Reply:
yes, no change

Reply:
OK thanks.